themes/ohcoe/static/js/scripts.js

```diff
@@ -1,5 +1,15 @@
 /* eslint-disable scanjs-rules/identifier_localStorage */
 $(function() {
+    /* Utility Methods */
+    function round(val){
+        return Math.round(val * 100) / 100;
+    }
+
+    function mean(arr){
+        return arr.reduce(function(acc, val){
```

Comment:
On page load, this error is being thrown: `Uncaught TypeError: Reduce of empty array with no initial value`.